content/en/blog/2022/how-to-get-started-with-opentelemetry-go.md

```diff
@@ -0,0 +1,420 @@
+---
+title: Go Web App Instrumentation
+linkTitle: Go App Instrumentation
+date: 2022-06-27
+author: Naveh Mevorach (Aspecto)
+canonical_url: https://www.aspecto.io/blog/opentelemetry-go-getting-started/
+---
+
+
+In this blog post, you will learn hands-on how to create and visualize traces with OpenTelemetry Go without prior knowledge.
+
+We will start with creating a simple to-do app that uses Mongo and the Gin framework. Then, we will send tracing data to Jaeger Tracing for visualization. You can find all the relevant files in this [Github repository](https://github.com/aspecto-io/opentelemetry-examples/tree/master/go).
+
+!["OpenTelemetry Go The Mandalorian"](https://www.aspecto.io/wp-content/uploads/2022/06/OpenTelemetry-Go-The-Mandalorian-2048x1406.png)
```

Comment:
@chalin, @austinlparker : should images in the blog/docs live here or is it okay to load them from a 3rd party URL?

Reply:
Ideally they should live in this repo, and (double) ideally they should be packaged as [page resources]. 

@NavehMevorach, if you add the images to this PR, I can organize them all into a [page bundle] for this post for you.

[page resources]: https://gohugo.io/content-management/page-resources/
[page bundle]: https://gohugo.io/content-management/page-bundles/

Reply:
@chalin Let me first regenerate the images and then I will add them to the PR

Reply:
@NavehMevorach - if you'd like I can repackage things into a page bundle after you're done and before the final publication.